2022-05-25 23:16:50,837 INFO    MainThread:16415 [wandb_setup.py:_flush():75] Loading settings from /home/rgundaka/.config/wandb/settings
2022-05-25 23:16:50,837 INFO    MainThread:16415 [wandb_setup.py:_flush():75] Loading settings from /central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/settings
2022-05-25 23:16:50,838 INFO    MainThread:16415 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-05-25 23:16:50,838 WARNING MainThread:16415 [wandb_setup.py:_flush():75] Could not save program above cwd: /groups/tensorlab/rgundaka/code/PINO/train_operator.py
2022-05-25 23:16:50,838 INFO    MainThread:16415 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': None, 'program': '/groups/tensorlab/rgundaka/code/PINO/train_operator.py'}
2022-05-25 23:16:50,838 INFO    MainThread:16415 [wandb_setup.py:_flush():75] Applying settings from <Settings {'_args': None, '_cli_only_mode': None, '_config_dict': None, '_cuda': None, '_debug_log': None, '_disable_meta': None, '_disable_stats': None, '_disable_viewer': None, '_except_exit': None, '_executable': None, '_internal_check_process': 8, '_internal_queue_timeout': 2, '_jupyter_name': None, '_jupyter_path': None, '_jupyter_root': None, '_os': None, '_python': None, '_require_service': None, '_runqueue_item_id': None, '_save_requirements': True, '_service_transport': None, '_start_datetime': None, '_start_time': None, '_tmp_code_dir': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None/tmp/code', '_unsaved_keys': None, 'allow_val_change': None, 'anonymous': None, 'api_key': '***REDACTED***', 'base_url': 'https://api.wandb.ai', 'code_dir': None, 'config_paths': None, 'console': 'auto', 'disable_code': None, 'disable_git': None, 'disabled': False, 'docker': None, 'email': None, 'entity': None, 'files_dir': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None/files', 'force': None, 'git_remote': 'origin', 'heartbeat_seconds': 30, 'host': None, 'ignore_globs': (), 'label_disable': None, 'launch': None, 'launch_config_path': None, 'log_dir': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None/logs', 'log_internal': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None/logs/debug-internal.log', 'log_symlink_internal': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/debug-internal.log', 'log_symlink_user': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/debug.log', 'log_user': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None/logs/debug.log', 'login_timeout': None, 'magic': None, 'mode': 'online', 'notebook_name': None, 'problem': 'fatal', 'program': None, 'program_relpath': None, 'project': None, 'quiet': None, 'reinit': None, 'relogin': None, 'resume': None, 'resume_fname': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/wandb-resume.json', 'root_dir': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test', 'run_group': None, 'run_id': None, 'run_job_type': None, 'run_name': None, 'run_notes': None, 'run_tags': None, 'sagemaker_disable': None, 'save_code': None, 'settings_system': '/home/rgundaka/.config/wandb/settings', 'settings_workspace': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/settings', 'show_colors': None, 'show_emoji': None, 'show_errors': True, 'show_info': True, 'show_warnings': True, 'silent': False, 'start_method': 'fork', 'strict': None, 'summary_errors': None, 'summary_warnings': 5, 'sweep_id': None, 'sweep_param_path': None, 'symlink': None, 'sync_dir': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None', 'sync_file': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None/run-None.wandb', 'sync_symlink_latest': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/latest-run', 'system_sample': 15, 'system_sample_seconds': 2, 'tmp_dir': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-None-None/tmp', 'username': None, '_console': <SettingsConsole.REDIRECT: 2>, '_jupyter': False, '_kaggle': False, '_noop': False, '_offline': False, '_quiet': None, '_show_info': True, '_show_warnings': True, '_show_errors': True, '_silent': False, '_strict': None, '_windows': False, 'is_local': False, 'run_mode': 'run', 'timespec': None, 'wandb_dir': '/central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/'}>
2022-05-25 23:16:50,845 INFO    MainThread:16415 [wandb_init.py:_log_setup():386] Logging user logs to /central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-20220525_231650-31cb1iin/logs/debug.log
2022-05-25 23:16:50,845 INFO    MainThread:16415 [wandb_init.py:_log_setup():387] Logging internal logs to /central/groups/tensorlab/rgundaka/code/PINO/CPINO/experiments/Darcy_cpino_batchsize_test/wandb/run-20220525_231650-31cb1iin/logs/debug-internal.log
2022-05-25 23:16:50,845 INFO    MainThread:16415 [wandb_init.py:init():420] calling init triggers
2022-05-25 23:16:50,845 INFO    MainThread:16415 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {'data': {'datapath': '/groups/tensorlab/rgundaka/code/data/piececonst_r421_N1024_smooth1.mat', 'n_sample': 1000, 'name': 'Darcy', 'nx': 421, 'offset': 0, 'sub': 7, 'total_num': 1024}, 'model': {'activation': 'gelu', 'competitive': True, 'fc_dim': 128, 'layers': [64, 64, 64, 64, 64], 'modes1': [20, 20, 20, 20], 'modes2': [20, 20, 20, 20]}, 'others': {'entity': 'rishigundakaram', 'group': 'Darcy-train', 'project': 'CPINO'}, 'train': {'base_lr': 0.001, 'batchsize': 1024, 'epochs': 25, 'f_loss': 1.0, 'lr_max': 0.01, 'lr_min': 0.01, 'milestones': [100, 150, 200], 'save_dir': '', 'save_name': 'darcy-cpino-5.pt', 'scheduler_gamma': 0.5, 'xy_loss': 0}}
2022-05-25 23:16:50,845 INFO    MainThread:16415 [wandb_init.py:init():471] starting backend
2022-05-25 23:16:50,846 INFO    MainThread:16415 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-05-25 23:16:50,895 INFO    MainThread:16415 [backend.py:ensure_launched():219] starting backend process...
2022-05-25 23:16:50,959 INFO    MainThread:16415 [backend.py:ensure_launched():225] started backend process with pid: 16542
2022-05-25 23:16:50,973 INFO    MainThread:16415 [wandb_init.py:init():480] backend started and connected
2022-05-25 23:16:51,039 INFO    MainThread:16415 [wandb_init.py:init():550] updated telemetry
2022-05-25 23:16:51,149 INFO    MainThread:16415 [wandb_init.py:init():581] communicating current version
2022-05-25 23:16:56,150 INFO    MainThread:16415 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-05-25 23:16:58,788 INFO    MainThread:16415 [wandb_init.py:init():624] starting run threads in backend
2022-05-25 23:17:02,239 INFO    MainThread:16415 [wandb_run.py:_console_start():1827] atexit reg
2022-05-25 23:17:02,240 INFO    MainThread:16415 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-05-25 23:17:02,242 INFO    MainThread:16415 [wandb_run.py:_redirect():1706] Redirecting console.
2022-05-25 23:17:02,271 INFO    MainThread:16415 [wandb_run.py:_redirect():1762] Redirects installed.
2022-05-25 23:17:02,271 INFO    MainThread:16415 [wandb_init.py:init():651] run started, returning control to user process
2022-05-25 23:17:03,028 INFO    MainThread:16415 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2022-05-25 23:17:03,030 INFO    MainThread:16415 [wandb_run.py:_restore():1769] restore
2022-05-25 23:17:05,281 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 8877
  total_bytes: 8877
}

2022-05-25 23:17:05,392 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 8877
  total_bytes: 8877
}

2022-05-25 23:17:05,539 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 8877
  total_bytes: 8877
}

2022-05-25 23:17:05,811 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 8877
  total_bytes: 14814
}

2022-05-25 23:17:05,912 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 8877
  total_bytes: 14814
}

2022-05-25 23:17:06,013 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 13617
  total_bytes: 14814
}

2022-05-25 23:17:06,115 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 14814
  total_bytes: 14814
}

2022-05-25 23:17:06,216 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 14814
  total_bytes: 14814
}

2022-05-25 23:17:06,318 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 14814
  total_bytes: 14814
}

2022-05-25 23:17:06,420 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 14814
  total_bytes: 14814
}

2022-05-25 23:17:07,059 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 14814
  total_bytes: 14814
}

2022-05-25 23:17:07,331 INFO    MainThread:16415 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 7
}
pusher_stats {
  uploaded_bytes: 14814
  total_bytes: 14814
}
local_info {
}

2022-05-25 23:17:08,555 INFO    MainThread:16415 [wandb_run.py:_append_files():2194] logging synced files
